addons/purchase/models/purchase.py

```diff
@@ -525,6 +533,14 @@ def _compute_amount(self):
                 'price_subtotal': taxes['total_excluded'],
             })
 
+    @api.multi
+    def _compute_tax_id(self):
+        for line in self:
+            fpos = line.order_id.fiscal_position_id or line.order_id.partner_id.property_account_position_id
```

Comment:
Although I agree on your comment, using the admin for the regular daily operations **is** a bad practice. Especially in multi-company environments, this is really asking for trouble since multi-company record rules are not applied.